src/OrbitGl/TimeGraphLayout.h

```diff
@@ -27,8 +27,10 @@ class TimeGraphLayout {
   float GetTrackContentBottomMargin() const { return track_content_bottom_margin_ * scale_; }
   float GetTrackContentTopMargin() const { return track_content_top_margin_ * scale_; }
   float GetTrackLabelOffsetX() const { return track_label_offset_x_; }
-  float GetSliderWidth() const { return slider_width_; }
-  float GetTimeBarHeight() const { return time_bar_height_; }
+  float GetSliderWide() const { return slider_wide_ * scale_; }
```

Comment:
I don't get the renaming, "GetSliderWidth()" is perfectly fine. Using the "wide" adjective instead of the "width" noun is weird.

Reply:
I think it is because it also used to express height. So I suggest "size" is better.

Reply:
I wanted to suggest a different name because we tend to use width to the x-distance between 2 points (and height to the y_distance), but I will revert it. I don't like size.

This value is:
- The y-size of the horizontal scrollbar
- The x-size of the vertical's one. 

So, I will rollback it.